vimeo-networking/src/main/java/com/vimeo/networking/model/VodItem.java

```diff
@@ -0,0 +1,220 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016 Vimeo
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.vimeo.networking.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.io.Serializable;
+import java.util.Date;
+
+/**
+ * This class represents the model for a Video On Demand (VOD) container
+ * Created by rigbergh on 4/25/16.
+ */
+public class VodItem implements Serializable{
+    public enum VodType {
+        FILM,
+        SERIES,
+        UNKNOWN;
+        private static String S_FILM = "film";
+        private static String S_SERIES = "series";
+
+        public static VodType from(@NotNull String type){
+            if(type.equalsIgnoreCase(S_FILM)){
+                return FILM;
+            } else if(type.equalsIgnoreCase(S_SERIES)){
+                return SERIES;
+            }
+            return UNKNOWN;
+        }
+
+        @Override
+        public String toString() {
+            if(this == FILM){
+                return S_FILM;
+            } else if(this == SERIES){
+                return S_SERIES;
+            }
+            return null;
+        }
+    }
+
+    private static class Publish implements Serializable{
+
+        private static final long serialVersionUID = -994389241935894370L;
+
+        @SuppressWarnings("unused")
+        @Nullable
+        @SerializedName("time")
+        public Date time;
+    }
+
+    private static final long serialVersionUID = 8360150766347816073L;
+
+    @SuppressWarnings("unused")
+    @Nullable
+    @SerializedName("name")
+    private String mName;
+
+    @SuppressWarnings("unused")
+    @Nullable
+    @SerializedName("description")
+    private String mDescription;
+
+    @SuppressWarnings("unused")
+    @Nullable
+    @SerializedName("type")
+    private String mType;
```

Comment:
Instead of String here, use ```VodType```, then have constructor params for the VodType enum. Then "type" will automatically be deserialized into a VodType.